buildspec/linuxTests.yml

```diff
@@ -29,7 +31,24 @@ phases:
                     npm run testCompile
                     npm run lint
                 fi
-            - xvfb-run npm test --silent
+            - |
+                {
+                    set -o pipefail
+                    _PROMISE_REJ_MSG='rejected promise not handled'
+                    mkfifo testout
+                    rm -rf testout-rej
+                    touch testout-rej
+                    (cat testout &)
+                    xvfb-run npm test --silent | tee testout | { >>testout-rej grep --line-buffered "$_PROMISE_REJ_MSG" || true; }
+                    if [ -s testout-rej ] ; then
+                        cat testout-rej | sort
+                        printf '\n\nERROR: Found "%s" %s times in test output (see above).\n\n' \
+                            "$_PROMISE_REJ_MSG" \
+                            "$(cat testout-rej | wc -l | tr -d ' ')"
+                        # TODO: fail the CI job
+                        # exit 1;
```

Comment:
For now, this is only informational. Can enable `exit 1` after we drive these down to zero. https://github.com/aws/aws-toolkit-vscode/pull/3814